app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -444,4 +445,15 @@ public static void addShortcut(
 
     ShortcutManagerCompat.requestPinShortcut(context, info, null);
   }
+
+  public static boolean appInstalledOrNot(String uri, PackageManager pm) {
```

Comment:
Please don't add things to Utils, as they will get lost, create a specific class for these that make it easier to find later when needed.